app/assets/javascripts/modules/orgs.js

```diff
@@ -13,6 +13,22 @@ angular.module('probedock.orgs', [ 'probedock.api', 'probedock.auth', 'probedock
         service.emit('refresh', service.organizations);
       },
 
+      getOrganization: function(orgId) {
+        var org =_.findWhere(service.organizations, { id: orgId });
+
+        if (_.isUndefined(org)) {
+          return api({
+            url: '/organizations/' + orgId
+          })
+          .then(function(response) {
```

Comment:
In general, the convention applied in the codebase so far is not to precede chained calls with new lines, except in special cases when there are many chained calls (e.g. states or module definitions).